app/__init__.py

```diff
@@ -3,6 +3,8 @@
 from flask_migrate import Migrate
 import os
 from dotenv import load_dotenv
+# from slack_sdk import WebClient
+# from slack_sdk.errors import SlackApiError
```

Comment:
since we aren't using these imports anymore, don't forget to get rid of them for the final project submission!

```suggestion

```